sys-kernel/coreos-modules/files/commonconfig-5.15

```diff
@@ -170,6 +170,7 @@ CONFIG_EDAC=m
 CONFIG_EFI=y
 CONFIG_EFIVAR_FS=y
 CONFIG_EFI_STUB=y
+CONFIG_EFI_GENERIC_STUB_INITRD_CMDLINE_LOADER=y
```

Comment:
Please keep the file contents sorted, so I think this line should go up one line. Or use the `sort_config.sh` script in the same directory. :)